src/lib.rs

```diff
@@ -183,7 +183,10 @@ pub mod signal;
 pub mod source;
 pub use source::*;
 mod time_val;
+#[macro_use]
 pub mod translate;
+pub mod gstring;
```

Comment:
You can keep the above `mod gstring;` btw if there's nothing that ever would have to be imported from there directly.